schemas/context.json

```diff
@@ -229,6 +229,23 @@
 				"nullable"
 			]
 		},
+		"copySubmissionAckPrimaryContact": {
+			"type": "boolean",
+			"default": false,
+			"validation": [
+				"nullable"
+			]
+		},
+		"copySubmissionAckAddress": {
+			"type": "array",
+			"default": [],
+			"items": {
+				"type": "string",
+				"validation": [
+					"email_or_localhost"
+				]
+			}
+		},
```

Comment:
We can add a `description` property to each of these, and it will be used in the REST API docs. Use the descriptions that I recommended above.